src/main/docker/bamboo/Dockerfile

```diff
@@ -5,9 +5,10 @@ LABEL description="Bamboo pre-configured for Artemis"
 USER root
 ENV DEBIAN_FRONTEND noninteractive
 
-RUN apt-get update
-RUN apt-get install -y maven
-RUN apt-get install -y software-properties-common apt-utils
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
why do we need to change this Dockerfile at all?
It also does not include dependencies to any other programming language

Reply:
in case we wanted to use an updated version of maven, we need a way to tell docker to skip the cache and use the defined version. This is why it's a good practice to name the version of dependencies. This technique is known as “cache busting” (https://docs.docker.com/develop/develop-images/dockerfile_best-practices/)